include/zephyr/devicetree.h

```diff
@@ -1082,6 +1145,49 @@
 #define DT_STRING_UPPER_TOKEN_BY_IDX(node_id, prop, idx) \
 	DT_CAT6(node_id, _P_, prop, _IDX_, idx, _STRING_UPPER_TOKEN)
 
+/**
+ * @brief Get a string array item value as an unquoted string content.
```

Comment:
Please make similar changes to this docstring as the ones suggested above.

Reply:
OK